### PR TITLE
Enhancement of UG

### PR DESCRIPTION
**Pull request by mamayuan**
Specified `INDEX` we use is 1-based.

Added a command summary table at the end.

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -450,5 +450,5 @@ testers are expected to do more *exploratory* testing.
 1. _{ more test cases …​ }_
 
 ## **Appendix: Planned enhancements**
-
+1. Add a boolean field to deleteCommand class to indicate whether user has entered duplicate indicies in the command. Display an extra message to remind the user of it.
```
Comment (Cloud7050) [inline docs/DeveloperGuide.md:453]: - Could probably change capitalisation of `DeleteCommand` and add formatting
- Typo in "indicies"
- Do we need further elaboration? If the planned enhancement does not address the feature flaw enough, testers may still re-report a flaw with the planned enhancement during the PE

File: docs/UserGuide.md [hunk 2]
```diff
@@ -52,6 +52,8 @@ Once you are familiarised, feel free to use the `clear` command to delete all th
 - Some commands take in parameters.
 e.g. in `add n/NAME`, the `add` command takes in an `n/` parameter.
 
+- `INDEX` used in commands is always 1-based.
```
Comment (Cloud7050) [inline docs/UserGuide.md:55]: I was thinking, if we address the other issue about having a labelled image explaining parts of the UI, then the topic of indices should already be made clearer.

But no harm adding more detail as well. However, should we merge this info into the part that explains indices instead?

image

Come to think of it, isn't it already pretty clear whether the index is 0 or 1-based from that explanation? 0 is not positive. To what extent do we need to elaborate, if at all?

There was an issue about this topic (#172), I think it was more so about the in-app output, but that output is meant to be short while the full details are in the UG. A bit like `git help`.

File: docs/UserGuide.md [hunk 3]
```diff
@@ -239,6 +241,26 @@ The displayed contact list is always automatically sorted in ascending alphabeti
 
 Your contacts get automatically saved to the file system after each successful command execution. There is no need to save manually.
 
+## Command Summary
```
Comment (Cloud7050) [inline docs/UserGuide.md:244]: For consistency, we could add a `---` hr above this new h2 section. We could also not capitalise the "s" in "Summary".

**Review by Cloud7050 — CHANGES_REQUESTED** [review]

**Comment by Cloud7050** [comment]
Btw, does this PR target any specific currently open issue(s)?

**Comment by mamayuan** [comment]
> Btw, does this PR target any specific currently open issue(s)?

The summary table is just something I saw in other team's UG. I think it's a good to have/ I remember one of the testers also mentioned to better include a summary but I cant find it now.

File: docs/UserGuide.md (hunk 2 above)
Comment (mamayuan) [reply docs/UserGuide.md:55]: Yea I agree. I think the tester overlooked this part. i have removed the additional clarification from the PR.

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (mamayuan) [reply docs/DeveloperGuide.md]: Yup changed it.

**Review by Cloud7050 — APPROVED** [review]
Approving if you are satisfied with the level of elaboration.

One thing I noticed, we could probably use the term "warning" as I think we will be adding more such warnings as planned enhancements. Ie "an additional message to remind the user" → "a warning in the output".

**Review by rayshawntan — APPROVED** [review]

**Pull request merged**